### PR TITLE
Playwright/dev mode

### PR DESCRIPTION
**Pull request by Arnaud97234**
## 🦒 Context (issues, jira)

Test UI - Playwright

## 💻  Description / Demo (image or video)
- Testnet currencies shouldn't be available in add account flow when developer mode is OFF
- User can enable developer mode from settings -> Experimental
- Testnet currencies should be available in add account flow when developer mode is ON
- User can add accounts from testnet currencies

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [ ] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

### CODE REVIEW

**Review by nabil-brn — DISMISSED** [review]
👌

File: playwright/specs/devmode.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const modal = new Modal(page);
+  const accountsPage = new AccountsPage(page);
+  const addAccountModal = new AddAccountModal(page);
+  const settingsPage = new SettingsPage(page);
+  const deviceAction = new DeviceAction(page);
+
+  for (const currency of currencies) {
+    await test.step(`when devMode OFF -> ${currency} shouldn't be available`, async () => {
+      await layout.goToAccounts();
+      await layout.goToAccounts();
```
Comment (elbywan) [inline playwright/specs/devmode.spec.ts:24]: Duplicate?

**Review by elbywan — DISMISSED** [review]
👌

File: playwright/specs/devmode.spec.ts (hunk 1 above)
Comment (nabil-brn) [reply playwright/specs/devmode.spec.ts:24]: I'll fix it in another PR, thank you!

**Review by nabil-brn — APPROVED** [review]

**Review by elbywan — APPROVED** [review]
👍

File: playwright/specs/devmode.spec.ts (hunk 1 above)
Comment (aulric-ledger) [reply playwright/specs/devmode.spec.ts]: My bad

**Pull request merged**